docs/pages/setup/reference/authentication.mdx

```diff
@@ -1,22 +1,21 @@
 ---
 title: Authentication options
-description: Teleport enterprise license file configuration parameters and requirements
+description: A reference for Teleport's authentication connectors
 ---
 
-Teleport uses the concept of "authentication connectors" to authenticate users
-when they execute [`tsh login`](./cli.mdx#tsh-login) command. There are three
-types of authentication connectors:
+Teleport uses the concept of **authentication connectors** to authenticate users
+when they execute the [`tsh login`](./cli.mdx#tsh-login) command. There are three
+types of authentication connectors.
 
 ## Local
 
 Local authentication is used to authenticate against a local Teleport user
-database. This database is managed by [`tctl users`](./cli.mdx#tctl-users-add)
+database. This database is managed by the [`tctl users`](./cli.mdx#tctl-users-add)
 command. Teleport also supports second-factor authentication (2FA) for the local
 connector. There are several possible values (types) of 2FA:
 
-- `otp` is the default. It implements [TOTP](https://en.wikipedia.org/wiki/Time-based_One-time_Password_Algorithm)
-  standard. You can use [Google Authenticator](https://en.wikipedia.org/wiki/Google_Authenticator)
-  or [Authy](https://www.authy.com/) or any other TOTP client.
+- `otp` is the default. It implements the [TOTP](https://en.wikipedia.org/wiki/Time-based_One-time_Password_Algorithm)
```

Comment:
I'm not sure if this is still accurate. 

@codingllama is OTP still the default MFA option?

Reply:
Correct, "otp" is the default for local second_factor settings. It makes sense, as there is nothing to configure for OTP ("webauthn" needs an RPID, at least).